### PR TITLE
Feature static asset deliver /2

### LINKED ISSUE #34033: Update What's New for .NET 9 to elevate the "Optimize static web asset delivery" section to be a top-level section
### Description

The [optimized handling of static web assets](https://learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-9.0?view=aspnetcore-9.0#optimize-static-web-asset-delivery) is one of the larger features in the .NET 9 release, so I'd like to elevate it to be a top-level section in the What's New doc instead of being a subsection under Miscellaneous.

### Page URL

https://learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-9.0?view=aspnetcore-9.0

### Content source URL

https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/release-notes/aspnetcore-9.0.md

### Document ID

4e75ad25-2c3f-b28e-6a91-ac79a9c683b6

### Article author

@Rick-Anderson

---
[Associated WorkItem - 340549](https://dev.azure.com/msft-skilling/Content/_workitems/edit/340549)

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #34033

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/migration/80-90.md](https://github.com/dotnet/AspNetCore.Docs/blob/3d7ff82a935f13ef7d4141c0ea554b7b59bf5e93/aspnetcore/migration/80-90.md) | [aspnetcore/migration/80-90](https://review.learn.microsoft.com/en-us/aspnet/core/migration/80-90?branch=pr-en-us-34055) |
| [aspnetcore/release-notes/aspnetcore-9.0.md](https://github.com/dotnet/AspNetCore.Docs/blob/3d7ff82a935f13ef7d4141c0ea554b7b59bf5e93/aspnetcore/release-notes/aspnetcore-9.0.md) | [aspnetcore/release-notes/aspnetcore-9.0](https://review.learn.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-9.0?branch=pr-en-us-34055) |

### CODE REVIEW

File: aspnetcore/release-notes/aspnetcore-9/includes/blazor.md [hunk 1]
```diff
@@ -232,6 +223,6 @@ The <xref:Microsoft.AspNetCore.Components.Forms.InputNumber%601> component now s
 
 <!-- UPDATE 10.0 Confirm or update -->
 
-Support for multiple Blazor Web Apps per server project will be considered for .NET 10 (November, 2025).
+Support for multiple Blazor Web Apps per server project will be considered for .NET 10 (November  2025).
```
Comment (guardrex) [inline aspnetcore/release-notes/aspnetcore-9/includes/blazor.md:226]: Typically, I'd expect the comma. Let's consider rolling it into the sentence. It reads a bit better ...

```suggestion
Support for multiple Blazor Web Apps per server project is under consideration for .NET 10 in late 2025.
```

**Review by guardrex — APPROVED** [review]
I just have a super NIT on one line.

File: aspnetcore/release-notes/aspnetcore-9.0.md [hunk 2]
```diff
@@ -11,9 +11,13 @@ uid: aspnetcore-9
 
 This article highlights the most significant changes in ASP.NET Core 9.0 with links to relevant documentation.
 
-This article has been updated for .NET 9 Release Candidate 2.
+## Static asset delivery optimization
 
-<!-- New content should be added to ~/aspnetcore-9/includes/newFeatureName.md files. This will help prevent merge conflicts in this file. -->
+Map Static Assets routing endpoint conventions, <xref:Microsoft.AspNetCore.Builder.StaticAssetsEndpointRouteBuilderExtensions.MapStaticAssets%2A>, is a new feature that optimizes the delivery of static assets in ASP.NET Core apps.
```
Comment (tdykstra) [inline aspnetcore/release-notes/aspnetcore-9.0.md:16]:
```suggestion
[MapStaticAssets routing endpoint conventions](xref:Microsoft.AspNetCore.Builder.StaticAssetsEndpointRouteBuilderExtensions.MapStaticAssets%2A) is a new feature that optimizes the delivery of static assets in ASP.NET Core apps.
```

File: aspnetcore/release-notes/aspnetcore-9.0.md [hunk 3]
```diff
@@ -11,9 +11,13 @@ uid: aspnetcore-9
...
+
+For more information on static asset deliver for Blazor apps, see <xref:blazor/fundamentals/static-files?view=aspnetcore-9.0>.
```
Comment (tdykstra) [inline aspnetcore/release-notes/aspnetcore-9.0.md:18]:
```suggestion
For information on static asset delivery for Blazor apps, see <xref:blazor/fundamentals/static-files?view=aspnetcore-9.0>.
```

File: aspnetcore/release-notes/aspnetcore-9/includes/blazor.md (hunk 1 above)
Comment (tdykstra) [reply aspnetcore/release-notes/aspnetcore-9/includes/blazor.md:226]: I would just change "(November 2025)" to "(late 2025)"

**Review by tdykstra — APPROVED** [review]
Looks good, I left a couple suggestions

**Pull request merged**